snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/tracker/EmitterTest.java

```diff
@@ -206,9 +206,9 @@ public void testUpdatingEmitterSettings() throws InterruptedException {
 
         assertTrue(emitter.getEmitterStatus());
         emitter.setHttpMethod(POST);
-        assertEquals("https://com.acme/i", emitter.getEmitterUri());
-        emitter.setEmitterUri("com/foo");
-        assertEquals("https://com.acme/i", emitter.getEmitterUri());
+        assertEquals("https://com.acme/com.snowplowanalytics.snowplow/tp2", emitter.getEmitterUri());
+        emitter.setEmitterUri("com.foo");
+        assertEquals("https://com.foo/com.snowplowanalytics.snowplow/tp2", emitter.getEmitterUri());
```

Comment:
I had to change the behavior of this test as it was testing that the network config doesn't change while emitter is running.